style.css

```diff
@@ -0,0 +1,730 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.first-bar {
+  display: none;
+}
+
+.seconde-bar {
+  display: none;
+}
+
+.Mobile-menu {
+  display: flex;
+  width: 100%;
+  justify-content: flex-start;
+  height: 100px;
+  align-items: center;
+  background-image: url(images/mur-texture-platre-gris-blanc-utiliser-comme-arriere-plan_1194-423737.jpg);
+}
+
+.Mobile-menu .humburger {
+  display: flex;
+  width: 6%;
+  padding: 5px;
+  font-size: 46px;
+  margin-left: 4%;
+}
+
+.minimenu {
+  display: flex;
+  background: linear-gradient(0deg, hsla(240, 15%, 45%, 1) 1%, hsla(240, 24%, 42%, 1) 27%, hsla(240, 29%, 41%, 1) 50%, hsla(240, 53%, 35%, 1) 74%, hsla(204, 77%, 76%, 1) 99%, hsla(240, 4%, 57%, 1) 100%);
+  width: 100%;
+  height: 100rem;
+  position: relative;
+  flex-direction: column;
+  margin-top: -100px;
+}
```

Comment:
- [ ] The mobile menu should follow the design of the page and use similar font families and colors: 
- Font families: Cocogoose, Lato.
- Color swatch: #272A31, #EC5242, #D3D3D3